mhapsite/mhap/models.py

```diff
@@ -45,6 +45,7 @@ def get_list_url(self):
 #http://stackoverflow.com/questions/1737017/django-auto-now-and-auto-now-add
 class ChatMessages(models.Model):
     message = models.CharField(max_length=255)
+    message_type = models.CharField(max_length=255, default="default")
```

Comment:
[Choice](http://stackoverflow.com/questions/24403075/django-choicefield)

James, I think its better to add a choice field rather than Char